scripts/build

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# execute all of the steps necessary to build the project
+set -e #-x
+
+usage() {
+  echo "usage: $0 [-h|--help] COMMAND"
+  echo ""
+  echo "[-h|--help]      print this help text"
+  echo ""
+  echo "COMMAND is one of the following:"
+  echo "- test           execute the tests (should be relatively fast)"
+  echo "- integration    execute the integration tests"
+  echo "- project        the default build command"
+  echo "- checkFormat    check formatting of files"
```

Comment:
Perhaps "lint" instead of "checkFormat"? Then repositories can choose to do any sort of static analysis that they want here.

Reply:
`lint` rolls off the tongue way easier (and even funnier to picture - 🤨) than `checkFormat` or `staticAnalysis` or ...

Reply:
OK, I will change `checkFormat` to lint. I think it is something we generally will want in our repos, as we do static analysis as part of ci.